.github/renovate.json

```diff
@@ -2,8 +2,11 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "dependencyDashboard": true,
   "suppressNotifications": ["prEditedNotification"],
-  "extends": ["config:base"],
+  "extends": ["config:recommended"],
   "labels": ["bot: dependencies"],
+  "semanticCommits": "disabled",
+  "separateMajorMinor": false,
+  "prHourlyLimit": 10,
```

Comment:
By default, renovate rate-limits itself to 2 PRs per hour so as to avoid "overwhelming" projects with a huge flood of PRs when they first enable renovate. With our current setup, it's unlikely that renovate would need to create more than 2 PRs an hour; but if it does feel the need, I think it should be able to create a few more at once